data/blog/electron-6-0.md

```diff
@@ -0,0 +1,122 @@
+ * session.clearHostResolverCache() [#17229](https://github.com/electron/electron/pull/17229)
+ * session.clearStorageData() [#17249](https://github.com/electron/electron/pull/17249)
+ * session.getBlobData() [#17303](https://github.com/electron/electron/pull/17303)
+ * session.getCacheSize()  [#17185](https://github.com/electron/electron/pull/17185)
+ * session.resolveProxy() [#17222](https://github.com/electron/electron/pull/17222)
+ * session.setProxy()  [#17222](https://github.com/electron/electron/pull/17222)
+ * webContents.hasServiceWorker() [#16535](https://github.com/electron/electron/pull/16535)
+ * webContents.printToPDF() [#16795](https://github.com/electron/electron/pull/16795)
+ * webContents.savePage() [#16742](https://github.com/electron/electron/pull/16742)
+ * webFrame.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+ * webFrame.executeJavaScriptInIsolatedWorld() [#17312](https://github.com/electron/electron/pull/17312)
+ * webviewTag.executeJavaScript() [#17312](https://github.com/electron/electron/pull/17312)
+
+These functions now have two forms, synchronous and Promise-based asynchronous:
+ * `dialog.showMessageBox()` [#17298](https://github.com/electron/electron/pull/17298)
+ * `dialog.showOpenDialog()` [#16973](https://github.com/electron/electron/pull/16973)
+ * `dialog.showSaveDialog()` [#17054](https://github.com/electron/electron/pull/17054)
+
+These functions now return Promises:
+ * `app.dock.show()` [#16904](https://github.com/electron/electron/pull/16904)
+
+### `Electron Helper (Render).app` and `Electron Helper (Plugin).app`
+
+There are now more Mac Helper application bundles.
+
+In order to enable the [hardened runtime](https://developer.apple.com/documentation/security/hardened_runtime_entitlements?language=objc), which restricts things like
+writable-executable memory and loading code signed by a different Team
+ID, special code signing entitlements must be granted to the Helper.
+
+To keep the capabilities scoped to the process types that require them, Chromium [added](https://chromium-review.googlesource.com/c/chromium/src/+/1627456) 
+two new variants of the Helper app: one for renderers (`Electron Helper (Render).app`) and one for plugins (`Electron Helper (Plugin).app`).
+
+Currently all three Helpers (the pre-existing `Electron Helper.app` in addition to the two new ones) will be code signed in the same way without
+any entitlements, although this may change in the future.
+
+## Breaking Changes
+
+### `app.allowRendererProcessReuse` option
+
+This release starts laying the groundwork for a future requirement that native Node modules loaded in the renderer process be either N-API or Context Aware. The reasons for this change are faster performance, stronger security, and reduced maintenance workload. Read the full details in [this issue](https://github.com/electron/electron/issues/18397). This change is expected to complete in Electron v11.
+
+Outline for change:
```

Comment:
This timeline is in the linked issue I don't think we should duplicate it here in case it needs to change